.github/workflows/ci_build.yml

```diff
@@ -211,6 +211,11 @@ jobs:
     - name: Update submodules
       run: |
         git submodule update --init
+    - name: Set up Java
+      uses: actions/setup-java@v2
+      with:
+        java-version: 17
+        distribution: "temurin"
```

Comment:
This looks good to me (although I don't know what `temurin` is).

The same lines will be needed in `sdk_android_build.yaml`, right?  Or does GFXR not use that any more?

Reply:
I misunderstood and thought you eant that it was only needed in ci_build.yaml.  It is now in both.  I tried `oracle` for the JDK and it didn't work (even though that used to).  Do you have an alternative?